src/entry.sh

```diff
@@ -0,0 +1,8 @@
+__enhancd::entry::git::root()
+{
+    if git rev-parse --is-inside-work-tree 2> /dev/null && [[ $(git rev-parse --show-toplevel) != $PWD ]]; then
```

Comment:
I suggest to quote the `$(git rev-parse --show-toplevel)` and `$PWD` to prevent issues with spaces in dir/file names

Reply:
According to the [bash manual](https://www.gnu.org/software/bash/manual/bash.html#Conditional-Constructs), 

> Word splitting and filename expansion are not performed on the words between the [[ and ]]

So i don't think it's necessary

Reply:
Cool! Didn't know that.